### DISCUSSION TITLE
Need some help with Nested Tables

### DISCUSSION THREAD

**Discussion by whiskeyhughes** [anvilistas/tabulator#144] [Q&A] [answered]
I would like to start by first saying thanks so much for creating the tabulator python wrapper, it has been incredible! 

I am trying to create a report from 3 separate Anvil App Tables. There is a table for Billed Info, Usage - Compute, and for Usage - Storage. I want to create a nested table, so that you can search for company name (each table has that column), and then see the following information:  
![image](https://github.com/anvilistas/tabulator/assets/24579922/82660819-017c-455c-9979-8329097b8ba0)

Does something like this seem feasible? Would it be better if I somehow combined the information into a single table? (I am currently pulling the info from 3 separate API calls.)

**Comment by antletem** [comment, marked as answer]
You might be able to achieve this using the [data tree](https://tabulator.info/docs/6.2/tree#structure) functionality. Of course this is not directly implemented in the wrapper, so you'll have to read the js documentation and play with it a bit, but in general I've successfully implemented a lot in the JS tabulator documentation.
